### PR TITLE
don't fetch all case IDs just to validate one

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: corehq/apps/zapier/views.py [hunk 1]
```diff
@@ -155,7 +156,9 @@ def post(self, request, *args, **kwargs):
         if not couch_user.is_member_of(domain):
             return HttpResponseForbidden("This user does not have access to this domain.")
 
-        if case_id not in CaseAccessors(domain).get_case_ids_in_domain(case_type):
+        try:
+            CaseAccessors(domain).get_case(case_id)
```
Comment (emord) [inline corehq/apps/zapier/views.py]: I think you may want to assert case_type as well

Comment (nickpell) [inline corehq/apps/zapier/views.py]: +1

Comment (snopoke) [inline corehq/apps/zapier/views.py]: turns out there was a test for that too, nice

Comment (snopoke) [inline corehq/apps/zapier/views.py]: 06f2a14

**Review by emord — APPROVED** [review]

**Pull request merged**